data.sql

```diff
@@ -4,4 +4,11 @@ INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,weight_kg)VALUE
 INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,weight_kg)VALUES ('Gabumon', '2018-11-15', 2, true, 8.0);
 INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,weight_kg)VALUES ('Pikachu', '2021-01-07', 1, false, 15.04);
 INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,weight_kg)VALUES ('Devimon', '2017-05-12', 5, true, 11.0);
+INSERT INTO animals(name,date_of_birth,escape_attempts,neutered,weight_kg) VALUES('Charmander','2020-02-08',0,false,-11);
+INSERT INTO animals(name,date_of_birth,escape_attempts,neutered,weight_kg) VALUES('Plantmon','2021-11-15',2,true,-5.7);
+INSERT INTO animals(name,date_of_birth,escape_attempts,neutered,weight_kg) VALUES('Squirtle','1993-04-02',3,false,-12.13);
+INSERT INTO animals(name,date_of_birth,escape_attempts,neutered,weight_kg) VALUES('Angemon','2005-06-12',1,true,-45);
+INSERT INTO animals(name,date_of_birth,escape_attempts,neutered,weight_kg) VALUES('Boarmon','2005-06-07',7,true,20.4);
+INSERT INTO animals(name,date_of_birth,escape_attempts,neutered,weight_kg) VALUES('Blosson','1998-10-13',3,false,7);
+INSERT INTO animals(name,date_of_birth,escape_attempts,neutered,weight_kg) VALUES('Ditto','2022-05-14',4,true,4);
```

Comment:
- [ ] Please make sure to insert the proper data as there's a typo in the name `Blosson` It should be `Blossom` the animal has also tried to escape so it should be `true` and it weighs `17` kgs not `7`
- [ ] The animal "Ditto" weighs `22.0`kgs so please fix this as well.